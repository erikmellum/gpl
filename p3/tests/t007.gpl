int a = 4;
int b = 5;
on space
{
    if (a != b)
        if (a < b)
            print("a < b");
        else print("a > b");
}
