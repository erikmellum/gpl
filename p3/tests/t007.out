gpl.cpp::main()
  input file(tests/t007.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(false)
  print_symbol_table(false)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).

If graphics was enabled gpl program would execute now.

gpl.cpp::main() done.
